content/chainguard/chainguard-images/videos/repro.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Reproducibility and Chainguard Images"
+linktitle: "Reproducibility and Chainguard Images"
+lead: ""
+description: "This video explains the importance of reproducibility and how to reproduce any
+Chainguard image from an attestation."
+type: "article"
+date: 2024-05-20T12:21:01+00:00
+lastmod: 2024-05-20T12:21:01+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 070
+toc: true
+---
+
+{{< youtube 0Qn2J89UEvI >}}
+
+## Clarification
+
+In this video we mention needing to keep copies of old APKs in order to be able to recreate images.
+This wasn't fully accuate -- in fact we do keep all our previously issued APKs, so you can build
```

Comment:
" -- " > "—" (use real em dash)

Reply:
Hugo actually takes care of this but I've changed it anyway.